CONTRIBUTORS.md

```diff
@@ -0,0 +1,12 @@
+# Cilium Contributors
+
+These are the wonderful people who have donated their time and effort to help
+make Cilium better for all of us to use. We wouldn't be where we are today
+without your contributions. Thank you! 💖
+
+New contributors should add their name to the bottom of this list on their first
+community contribution. If you are a contributor in other ways other than code, please
```

Comment:
Maybe add a note to explain that code contributors will automatically be added on new releases? I don't want to encourage code contributors to open new PRs to update this file when this is already handled automatically